deb/openmediavault/srv/salt/omv/deploy/apt/files/etc-apt-sources_list_d-openmediavault_list.j2

```diff
@@ -1,19 +1,17 @@
-{% set repo_url = salt['pillar.get']('default:OMV_APT_REPOSITORY_URL', 'http://packages.openmediavault.org/public/') -%}
-deb {{ repo_url }} {{ pillar['productinfo']['distribution'] }} main
-deb https://openmediavault.github.io/packages/ {{ pillar['productinfo']['distribution'] }} main
-# deb http://downloads.sourceforge.net/project/openmediavault/packages/ {{ pillar['productinfo']['distribution'] }} main
+{% set public_repo_url = salt['pillar.get']('default:OMV_APT_REPOSITORY_URL', 'http://packages.openmediavault.org/public/') -%}
+{% set packages_repo_url = salt['pillar.get']('default:OMV_APT_PACKAGES_REPOSITORY_URL', 'https://openmediavault.github.io/packages/') -%}
```

Comment:
I thing `OMV_APT_ALT_REPOSITORY_URL` is better because it is the fallback/alternative OMV repository URL. Please use a `{%-` at the beginning of the line.
```suggestion
{%- set public_alt_repo_url = salt['pillar.get']('default:OMV_APT_ALT_REPOSITORY_URL', 'https://openmediavault.github.io/packages/') -%}
```

Reply:
Could you please rename the variable to `OMV_APT_ALT_REPOSITORY_URL`. This name will reflect the use better than `OMV_APT_PACKAGES_REPOSITORY_URL` because `OMV_APT_ALT_REPOSITORY_URL` is an alternative repository for `OMV_APT_REPOSITORY_URL` in case the one of them is down.

Reply:
updated